domains/certificates/Query.API/API.UnitTests/MeasurementsSyncer/MeasurementsSyncerWorkerTest.cs

```diff
@@ -43,8 +43,8 @@ public class MeasurementsSyncerWorkerTest
     public MeasurementsSyncerWorkerTest()
     {
         var measurementSyncMetrics = Substitute.For<MeasurementSyncMetrics>();
-        var syncService = new MeasurementsSyncService(_syncServiceFakeLogger, _fakeSlidingWindowState, _fakeClient, new SlidingWindowService(measurementSyncMetrics),
-            new MeasurementSyncMetrics(), _fakeMeasurementPublisher, _fakeMeteringPointsClient);
+        var syncService = new MeasurementsSyncService(_syncServiceFakeLogger, _fakeSlidingWindowState, _fakeClient, new SlidingWindowService(measurementSyncMetrics, Options.Create(_options)),
```

Comment:
minor. / nitpick / don't need to fix.

You could have:
_options = Options.Create();
so the options is already the options (if that makes sense). So you don't have to create them several times in same test.

Reply:
Like this?: https://github.com/Energinet-DataHub/energy-origin/pull/1575/commits/bcf5a9673633def2b3092a30559c1717cb968383

Reply:
yes